tests/cicd/main_build.yml

```diff
@@ -258,7 +262,7 @@ jobs:
 
       # check test set performance
       set -e
-      python ../../../../tests/cicd/src/check_performance.py --infile metrics_test_patch_deconvnet_no_depth.json --test
+      # python ../../../../tests/cicd/src/check_performance.py --infile metrics_test_patch_deconvnet_no_depth.json --test
```

Comment:
similar comment to enable

Reply:
as discussed, I added a "TODO" to enable this after investigation!